pkg/controller/gslb/internal/depresolver/depresolver_config.go

```diff
@@ -17,6 +18,7 @@ func (dr *DependencyResolver) ResolveOperatorConfig() (*Config, error) {
 		dr.config = &Config{}
 		// set predefined values when not read from the environment variables
 		dr.config.ReconcileRequeueSeconds, _ = env.GetEnvAsIntOrFallback(reconcileRequeueSecondsKey, 30)
+		dr.config.ClusterGeoTag = env.GetEnvAsStringOrFallback(clusterGeoTagKey, "unset")
```

Comment:
just a comment: geotag don't need to be validated against values i.e. 

- crazy symbols ".?:\\!",  
- whitespace(s) "    "
- any size limit criteria - "dksjdkdskjsdjsdksdkjk sdlsfhldfh kdjfh kjhdkjfhdk hgdjk ghkjgh . fhjhfhlkhjhkjhkhgjklhjg....

Reply:
To be honest I do not really see a reason to prescribe limitation on geo tag. Its contents craziness level is up to the user imho. @somaritane thoughts ?

Reply:
I mean, world wide users are really crazy and when invalid value is entered it fails during runtime on place where nobody expects with message which is not bounded to an invalid value (i.e. If I put -1 into TTL it could fail during runtime with random network error instead of clear "invalid TTL configuration" on the start ).

Should be this allowed `Central Europe http://cz.eu/Prague?env=local` ?

Reply:
it's actually valid point as we construct ns dns names using the geo tag. Will look into it